advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -198,14 +198,20 @@ private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.Vulne
         }
     }
 
-    return org.ossreviewtoolkit.model.Vulnerability(
-        id = id,
-        references = references.map {
-            VulnerabilityReference(
-                url = URI.create(it.url.trim()),
-                scoringSystem = scoringSystem,
-                severity = severity,
-            )
+    val references = references.mapNotNull {
+        val url = try {
+            URI.create(it.url.trim())
```

Comment:
I propose to make use of our `String.toUri()` extension function instead. Something along the line of

    it.url.trim().toUri().onFailure { log... }.getOrNull()